components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -719,149 +718,20 @@ <h1 class="can-collapse defclose" data-name="details">
         
     <!-- ANNOTATIONS -->
 
+        <!-- templates are loaded based on config -->
+        {% for t in ome.metadata_pane|dictsort:"index" %}
```

Comment:
I was also thinking if index shouldn't be 10, 20, 30,... that will help to shovel them in between easier